src/vs/workbench/contrib/search/browser/searchActions.ts

```diff
@@ -827,8 +828,9 @@ export const focusSearchListCommand: ICommandHandler = accessor => {
 	});
 };
 
-function getElementsToOperateOnInfo(viewer: WorkbenchObjectTree<RenderableMatch, void>, currElement: RenderableMatch): RenderableMatch[] {
-	let elements: RenderableMatch[] = viewer.getSelection().filter((x): x is RenderableMatch => x !== null);
+
+function getElementsToOperateOnInfo(viewer: WorkbenchObjectTree<RenderableMatch, void>, currElement: RenderableMatch, sortConfig: ISearchConfigurationProperties): RenderableMatch[] {
+	let elements: RenderableMatch[] = viewer.getSelection().filter((x): x is RenderableMatch => x !== null).sort((a, b) => searchComparer(a, b, sortConfig.sortOrder));
```

Comment:
You asked me about sorting, I thought you were asking about the debug session picker 😅  (it had just come up in standup). What order does getSelection return?

Reply:
it returns the order that they select the items, I believe